support-services/src/main/scala/com/gu/support/promotions/PromotionTerms.scala

```diff
@@ -18,9 +19,9 @@ case class PromotionTerms(
 
 object PromotionTerms {
 
-  import com.gu.support.encoding.CustomCodecs.{encodeDateTime, decodeDateTime}
+  implicit val encodeDateTime: Encoder[DateTime] = Encoder.encodeLong.contramap(_.getMillis)
```

Comment:
client side needs a Long timestamp rather than ISO, because the most reliable way across all browsers is millis since 1970.